config/webpack.test.js

```diff
@@ -18,6 +18,9 @@
 const path = require('path');
 const webpack = require('webpack');
 
+/** A regular expression used to replace Firestore' platform specific modules. */
```

Comment:
I expanded the comment and changed the RegExp to match all files. This is cleaner, but I initially wanted to keep it more restricted because I am afraid that it will start matching files in other platforms. Unfortunately, I haven't found a way to restrict it to just "packages/firestore" since the paths that are used are the relative import paths we specify in our sources. 

As a general heads up - I debated getting rid of the platform.ts file before, but kept it as is during the first review. I have since moved all implementations to specific "category" files (base64.ts, serializer.ts, dom.ts, etc). These seems cleaner that having two files that are outliers.